File: UsingSwiftClassesInObjC/UsingSwiftClassesInObjC/ViewController.m
//
//  ViewController.m
//  UsingSwiftClassesInObjC
//
//  Created by Flori on 22.01.15.
//  Copyright (c) 2015 Flori & Andy. All rights reserved.
//

#import "ViewController.h"
#import "UsingSwiftClassesInObjC-Bridging-Header.h"


@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    PureSwift * swiftObject = [[PureSwift alloc] init];
    [swiftObject myFunc];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: 18_MagicalRecord/11_MagicalRecord/ViewController.swift
//
//  ViewController.swift
//  11_MagicalRecord
//
//  Created by Andreas Wittmann on 29/11/15.
//  Copyright © 2015 🐼 + 🙉. All rights reserved.
//

import UIKit

class ViewController: UIViewController, NSFetchedResultsControllerDelegate, UITableViewDelegate, UITableViewDataSource {

    @IBOutlet weak var tableView: UITableView!
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

